proto/buf.gen.pulsar.yaml

```
@@ -0,0 +1,17 @@
+    default: cosmossdk.io/api
+    except:
+      - buf.build/googleapis/googleapis
+      - buf.build/cosmos/gogo-proto
+      - buf.build/cosmos/cosmos-proto
+plugins:
+  - name: go-pulsar
+    out: ../api
+    opt: paths=source_relative
+  - name: go-grpc
+    out: ../api
+    opt: paths=source_relative
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)